### PR TITLE
Refresh

### PR DESCRIPTION
**Pull request by martinpaljak**
- update dependencies, tools, plugins
- small code fixes to please (and clean) spotbugs
- obsoletes #89

### CODE REVIEW

File: tool/src/main/java/com/fidesmo/fdsm/Main.java [hunk 1]
```diff
@@ -272,7 +272,7 @@ public static void main(String[] argv) {
 
                 // Allows to run with any card
                 if (args.has(OPT_QA)) {
-                    String number = Integer.toString(new Random().nextInt(900000) + 100000).substring(0, 6);
+                    String number = Integer.toString(ThreadLocalRandom.current().nextInt(900000) + 100000).substring(0, 6);
```
Comment (sergkh) [inline tool/src/main/java/com/fidesmo/fdsm/Main.java:275]: IMO it was better previously :)

**Review by sergkh — APPROVED** [review]
Wow! That's a great update!

File: tool/src/main/java/com/fidesmo/fdsm/Main.java (hunk 1 above)
Comment (martinpaljak) [reply tool/src/main/java/com/fidesmo/fdsm/Main.java:275]: Did not know about ThreadLocalRandom (as normally only use SecureRandom) but this happens to please spotbugs as well.

Comment (sergkh) [reply tool/src/main/java/com/fidesmo/fdsm/Main.java:275]: I mean it's `main` thread so leaving random attached to it makes little sense as it's never re-used. But if it pleases spotbugs that's fine.

Comment (martinpaljak) [reply tool/src/main/java/com/fidesmo/fdsm/Main.java:275]: Focus here is "pleases spotbugs", yes.

**Comment by JSantosP** [comment]
Is there any blocker for merging this? Otherwise, @martinpaljak please feel free to do so

**Pull request merged**